lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -1191,6 +1198,56 @@ func networkFromContainerInfo(vc *viccontainer.VicContainer, info *models.Contai
 		Networks: make(map[string]*dnetwork.EndpointSettings),
 	}
 
+	shortCID := vc.ContainerID[0:ShortIDLen]
+
+	// Fill in as much info from the endpoint struct inside of the ContainerInfo.
+	// The rest of the data must be obtained from the Scopes portlayer.
+	for _, ep := range info.Endpoints {
+		netEp := &dnetwork.EndpointSettings{
+			IPAMConfig:          nil, //Get from Scope PL
+			Links:               nil,
+			Aliases:             nil,
+			NetworkID:           "", //Get from Scope PL
+			EndpointID:          ep.ID,
+			Gateway:             ep.Gateway,
+			IPAddress:           "",
+			IPPrefixLen:         0,  //Get from Scope PL
+			IPv6Gateway:         "", //Get from Scope PL
+			GlobalIPv6Address:   "", //Get from Scope PL
+			GlobalIPv6PrefixLen: 0,  //Get from Scope PL
+			MacAddress:          "", //Container endpoints currently do not have mac addr yet
+		}
+
+		// This data may not be valid if DHCP is used.  Get from Scope PL in that case.
+		if ep.Address != "" {
+			addrs := strings.Split(ep.Address, "/")
```

Comment:
On the network that is returned, e.g.
```go
_, n, err := net.ParseCIDR("10.10.10.10/24")
ones, _ := n.Mask.Size() // this would be the prefix length
```